packages/jellyfish-api-core/__tests__/category/blockchain/getChainTxStats.test.ts

```diff
@@ -0,0 +1,90 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { Testing } from '@defichain/jellyfish-testing'
+
+describe('ChainTxStats', () => {
+  const testing = Testing.create(new MasterNodeRegTestContainer())
+
+  beforeAll(async () => {
+    await testing.container.start()
+  })
+
+  afterAll(async () => {
+    await testing.container.stop()
+  })
+
+  it('should getChainTxStats for selected interval', async () => {
+    const blockHash = await testing.misc.waitForBlockHash(3)
+    const stats = await testing.rpc.blockchain.getChainTxStats(2, blockHash)
+
+    expect(stats.time).toBeGreaterThan(1550000000)
+    expect(stats.txcount).toStrictEqual(12)
+    expect(stats.window_final_block_hash.length).toStrictEqual(64)
```

Comment:
i think we can compare this against `blockHash` since we already have this info?